Modules/bookclass.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable class-methods-use-this */
+/* eslint-disable no-restricted-globals */
```

Comment:
- Please remove all disabled linters, remember that linters help to ensure that we write code that is clean and easy to maintain 😄 
HINT: The first disabled linter wants you to use the `this` keyword in your class methods, if you do not require the `this` keyword in your class method, use the `static` keyword before naming your class e.g ``` static storeBook () { ... } ```